TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+    "\\end{align}\n",
+    "\n",
+    "2. Discuss the above formula, explaining how covariance is very similar to variance, except with x and y instead, and that correlation is the 'covariance between x and y scaled by their standard deviations'.\n",
+    "\n",
+    "### Exercise\n",
+    "\n",
+    "| | |\n",
+    "|-|-|\n",
+    "|Let's play a little game then...<br><br>CLICK ME if you dare:<br>https://istics.net/Correlations/<br><br><br>Can you guess which plot<br>corresponds to which<br>correlation coefficient?<br><br><br><br><br><br><br><br><br>|<img src=\"im/7/guess_corr.png\" style=\"width:800px\">|\n",
+    "\n",
+    "### Short discussion\n",
+    "\n",
+    "- Will 'scaling' the independent or dependent variable have any effect on the correlation?\n",
+    "<br><br>\n",
+    "- Do not overestimate the usefulness of correlation, it can only tell us how tightly our data points are packed around a potential 'best-fit' or 'average' line.\n",
+    "<br><br>\n",
+    "- What matters mostly is the absolute value of the correlation, the sign only tells us the empirical slope of the data\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47674cd0",
+   "metadata": {},
+   "source": [
+    "## The Linear Regression Model [20 minutes]\n",
+    "|<img src=\"https://mobiledevmemo.com/wp-content/uploads/2012/10/w5449egf111.gif\" alt=\"\" style=\"width: 450px;\"/>|"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c9e2a651",
+   "metadata": {},
+   "source": [
+    "- The key idea in our understanding of linear regression is that we can attribute a linear relationship to a particular dataset.\n",
+    "<br><br>\n",
+    "- (Continuous) Outcome/Response/Dependent variable: $\\;\\; Y_i$\n",
+    "<br><br>\n",
+    "- Explanatory/Independent variable/Covariate/Feature/Predictor: $\\;\\; x_i$\n",
+    "<br><br>\n",
+    "- Our model has two parameters - **Intercept and Slope** - which we use to describe a relationship between $\\; Y_i$ and $\\; x_i$\n",
+    "\n",
+    " $$Y_i = \\beta_0 + \\beta_1 x_i + \\epsilon_i$$ \n",
+    "\n",
+    "- $\\epsilon_i$ is the **Noise** or the **Error** in our model which follows a normal distribution."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3083ba3e",
+   "metadata": {},
+   "source": [
+    "### Fitted Model\n",
+    "\n",
+    "- When we actually fit a linear model to a dataset, we obtain the following equation which contains estimates of what the slope and intercept of the line should be.\n",
+    "<br><br>\n",
+    "  $$\\hat y_i = \\hat \\beta_0 + \\hat \\beta_1 x_i$$ \n",
+    "  <br><br>\n",
+    "- Discuss how the above equation differs from  $Y_i = \\beta_0 + \\beta_1 x_i + \\epsilon_i$\n",
+    "<br><br>\n",
+    "- Discuss how **'Residuals'** $\\hat \\epsilon_i = y_i - \\hat y_i = y_i - \\hat \\beta_0 + \\hat \\beta_1 x_i $ differ from error terms.\n",
+    "  \n"
```

Comment:
This is the right appropriate content for the Oct 25 TUT, and I have repurposed it as such